charts/prometheus-blackbox-exporter/templates/serviceaccount.yaml

```
@@ -2,12 +2,11 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ template "prometheus-blackbox-exporter.serviceAccountName" . }}
+  name: {{ include "prometheus-blackbox-exporter.serviceAccountName" . }}
   labels:
-    app.kubernetes.io/name: {{ include "prometheus-blackbox-exporter.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/managed-by: {{ .Release.Service }}
-    helm.sh/chart: {{ include "prometheus-blackbox-exporter.chart" . }}
+    {{- include "prometheus-blackbox-exporter.labels" . | nindent 4 }}
+  {{- with .Values.serviceAccount.annotations }}
   annotations:
-{{ toYaml .Values.serviceAccount.annotations | indent 4 }}
-{{- end -}}
+    {{- toYaml . | nindent 4 }}
```

It depends what policy we want:
- all `{{ something }}` not indented
- follow logical indentation

If first:
```suggestion
{{ toYaml . | indent 4 }}
```

---

This comes also from "helm create" and imho it's nice using the same way as for the labels.
But i can live with the other version too :)

---

let's stick with it then!